RUFAS/classes.py

```diff
@@ -152,14 +153,21 @@ def __init__(self, data, weather_file):
 
             # compares actual size of the file to expected size
             if file_line - 1 != expected_w_file_size + 1:
-                #TODO add warnings to warning pool for each of these
-                print("Start and end dates of the Weather CSV file do not match the size.")
+                dates_not_matching_size = "Start and end dates of the Weather CSV file" \
+                    " do not match the size."
```

Comment:
Question for my own learning: Is this preferred way to break up along string? I thought you needed a `+` to concatenate two strings with python, as in:
```suggestion
                dates_not_matching_size = "Start and end dates of the Weather CSV file" + \
                    " do not match the size."
```

Reply:
Great question. I've been using it as a line continuation character like in Niko's code here (since it seems equivalent to using the '+' operator), but it would be good to know if there's edge cases where it could lead to problems.

Reply:
I was unsure about the + being necessary for the newline continuation of a string but it appears like it makes no difference:
<img width="718" alt="Screenshot 2022-12-06 at 1 36 27 PM" src="https://user-images.githubusercontent.com/70217952/205996059-ef27c3e8-2a00-4d29-8820-06d86d037e85.png">

Quick addendum via Pooya -

"+ only works for strings, \ is used to breakdown lines that cannot be broken down using other techniques. Using + \ adds unnecessary computation overhead and should be avoided. Using one of them is enough"

So I think we should keep it with \ alone.

Reply:
Yeah, that makes good sense to me and is useful to know! Thanks for the clarification.